qemu-tests/src/bin/eddsa_verify_on_target/main.rs

```diff
@@ -45,6 +45,11 @@ fn main () -> ! {
     loop { continue; }
 }
 
+fn fail() {
+    debug::exit(debug::EXIT_FAILURE);
+    loop { continue; }
```

Comment:
I don't think this is needed here (in my local changes, I just inline did the `debug::exit(debug::EXIT_FAILURE)`).

Reply:
I remember having issues without the endless loop after exit() once when running something on real hardware instead of `qemu`. Could be that it was with C code and therefore a different runtime. I have to check again.